migrations/2018_01_30_222800_create_users_table_preferences_columns.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('users', function (Blueprint $table) {
+            $table->boolean('disclose_online')->default(false);
```

Comment:
This too should be prefixed with `is_` according to the convention. Perhaps `is_online_public`, or maybe `is_last_seen_public` to associate it with `last_seen_time`...

Reply:
I like the `is_last_seen_public`. Let's go for that.